src/molecules/Carousel.tsx

```diff
@@ -28,6 +28,7 @@ export const Carousel: React.FC<CarouselProps> = ({ images }) => {
       >
         {images.map((image, index) => (
           <img
+            alt="productImg"
```

Comment:
Does product have a name/title? Consider using it here instead, so each product has a unique `alt` text.